proto/config/compound_devconfig.proto

```diff
@@ -8,15 +8,27 @@ option go_package  = "github.com/lf-edge/eve-api/go/config";
 option java_package = "org.lfedge.eve.config";
 
 import "auth/auth.proto";
+import "config/devconfig.proto";
 import "profile/local_profile.proto";
 
+message CompoundEdgeDevConfigRequest {
+  // Value of the field `seq` from the last received and successfully processed
```

Comment:
Nit (and I know whether we have the same wording in the LPS  API) but "processed" might not be what we do here.
Taking the most extreme example, if the LOC sends the poweroff command, then "processed" would mean that it sends the seq_ack for that command when the device is in fact powered off!

It has received it and will act on it (with sufficient persistence in EVE to act later even in precense of some disruption - which is why we persist e.g., the purge counter for an app instance to remember to do that after a power outage if we didn't get it done before the power went out.)

Reply:
I do not think that "command processing" implies "command completion". Command is processed and then in the future completed. I can add a few clarification words about it. What's important, that "processed" means "commands are persistent now in the file", which means that won't be lost, so the `ack_seq` can be safely set to the `seq`.

Reply:
Agreed.